CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
@dsogari are your other changes not compatible with CMake version 2.8.7? Your PR could be updated to revert this particular change if so.